doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+will need a robotic arm and a camera.
+
+If you haven't already done so, be sure to complete the steps in `Getting Started
+<../getting_started/getting_started.html>`_. Also, set your arm up to work with MoveIt (as described in the `Setup
+Assistant Tutorial <../setup_assistant/setup_assistant_tutorial.html>`_).
+
+This tutorial also requires a camera, publishing images and a ``sensor_msgs/CameraInfo`` topic with good intrinsic
+calibration parameters and an accurate coordinate frame. (Conduct an intrinsic camera calibration by using the
+`camera_calibration <http://wiki.ros.org/camera_calibration>`_ package, if necessary.)
+
+Clone and Build the MoveIt Calibration Repo
+-------------------------------------------
+In your workspace ``src`` directory, clone MoveIt Calibration::
+
+  git clone git@github.com:ros-planning/moveit_calibration.git
+
+Then, make sure you have the appropriate dependencies and build the package::
+
+  rosdep install -y --from-paths . --ignore-src --rosdistro melodic
+  catkin build
+  source devel/setup.sh
+
+Launch RViz and Load Calibration Plugin
+---------------------------------------
+Launch the appropriate MoveIt demo for your robot. For instance, ``roslaunch panda_moveit_config demo.launch``.
+In the RViz "Panels" menu, choose "Add New Panel":
+
+.. image:: images/choose_new_panel.png
+
+Then, select the "HandEyeCalibration" panel type:
+
+.. image:: images/add_handeye_panel.png
+
+The panel will be added with the "Target" tab active.
+
+Create and Print a Target
+-------------------------
+Now we will create a visual calibration target. This target has distinctive patterns that are easy to identify in the
+image data, and by providing a measurement of the target size, the pose of the target in the camera's coordinate frame
+can be estimated. When conducting a hand-eye calibration, we do not need to know the target's precise location--as long
+as the target is stationary in the robot's base frame, the hand-eye calibration can be estimated from a sequence of 5 or
+more poses.
+
+In the "Target Params" section of the "Target" tab, we will use the default target parameters:
+
+- **markers, X**: 3
+- **markers, Y**: 4
+- **marker size (px)**: 200
+- **marker separation (px)**: 20
+- **marker border (bits)**: 1
+- **ArUco dictionary**: DICT_5X5_250
+
+Press the "Create Target" button to create the target image:
+
+.. image:: images/aruco_target_handeye_panel.png
+
+Save the target image using the "Save Target" button, and print out the image. Feel free to experiment with the target
+parameters to see how they affect the target, but be sure to remember the parameters used for the target you print--you
+will need to input the same parameters for the target to be recognized.
+
+The target must be flat to be reliably localized by the camera. Laying it on a flat surface is sufficient, or it can be
+mounted to a board. Measure the marker width (the outside dimension of one of the black squares), as well as the
```

Comment:
Just to confirm, is it clear in the plugin what this "marker width" refers to? It should have a tooltip otherwise.

Reply:
Yes, a tooltip would be helpful. It's an open issue: ros-planning/moveit_calibration#16.